split.Console/Models/Game.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace split.Console.Models
+{
+    public class Game
+    {
+        public Guid Id;
+        public string Code;
+        public GameType Type;
+        public List<Player> Players;
+        public GameState State;
+
+        public Game(GameType type)
+        {
+            Code = GenerateRoomCode();
+            Id = new Guid(Code);
```

Comment:
I need to read the documentation a little for it. My question is there another way to generate a 4 alphanumeric character string other than the way I did it in the method. I agree with the NewGuid approach